docs/jps_guide/titlepage.tex.in

```diff
@@ -0,0 +1,29 @@
+%% this file is generated automatically. All manual changes will be deleted.
+\newcommand{\HRule}{\rule{\linewidth}{0.5mm}}
+\begin{titlepage}
+\begin{center}
+
+\includegraphics[width=0.75\textwidth]{../images/jupedsim_small.png} \\ [1cm]
+
+\textsc{\LARGE Jülich Pedestrian Simulator} \\ [1.5cm]
+
+\textsc{\small Forschungszentrum Jülich GmbH} \\ [0.5cm]
+
+\HRule \\ [0.4cm]
+{ \huge \bfseries User's Guide} \\ [0.4cm]
+
+
+\HRule \\ [1.5cm]
+
+\end{center}
+%>>>>>>>>>>>>>
```

Comment:
What does this line mean? Is this a legacy from an old git merge process?

Reply:
A comment to visualy split the first part from the second.

Reply:
I get it.